gatsby-config.js

```diff
@@ -29,10 +30,8 @@ module.exports = {
       }
     },
     {
-      resolve: 'gatsby-plugin-postcss',
-      options: {
-        postCssPlugins: [require('tailwindcss')('./tailwind.config.js')]
-      }
+      resolve: `gatsby-plugin-purgecss`,
```

Comment:
We are having some issues with this and TS. I needed to remove it since some styles were not being applied to production. I created a ticket to address this later 

Ihttps://app.asana.com/0/1159164196409129/1192037358959241/f